oo-workshop/mobprogrammingexercises/src/test/java/com/example/domainmodeling/anemic/ShoppingCartTest.java

```diff
@@ -0,0 +1,67 @@
+package com.example.domainmodeling.anemic;
+
+import com.example.domainmodeling.anemic.solution.InvalidCustomerException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class ShoppingCartTest {
+
+    private ShoppingCart tested;
+    private Customer validTestCustomer;
+    private List<CartItem> testItems;
+
+    @Before
+    public void setup() {
+        validTestCustomer = new Customer(UUID.randomUUID().toString(), "testCustomer");
+        testItems = new ArrayList<>();
+    }
+
+    @Test(expected = InvalidCustomerException.class)
+    public void givenAShoppingCartIsToBeCreated_whenACustomerDoesNotExist_ThrowsInvalidCustomerException() throws InvalidCustomerException {
+//        //arrange
```

Comment:
@jj-stamp-concord might just be missing the point but why are we keeping commented out code?

Reply:
@headl033 I had commented them out so I could run the solution tests. I had added these tests to run red and give some direction to the refactors needed in the exercise as well as give test examples for people going through the workshop with maybe less java experience but the unimplemented methods kept the solution test file from running.